src/sql/workbench/browser/modal/dialogHelper.ts

```diff
@@ -22,6 +21,15 @@ export function appendRow(container: HTMLElement, label: string, labelClass: str
 	}
 	const labelContainer = append(append(rowContainer, $(`td.${labelClass}`)), $('div.dialog-label-container'));
 	labelContainer.style.display = 'flex';
+
+	if (title) {
+		if (!titleMaxWidth) {
+			// Set max width of title to be 100 by default.
+			titleMaxWidth = 100;
```

Comment:
For now please only wrap the string if a max width is passed in so that the default behavior doesn't change. You can follow up with making the default 100 for all titles later on if you wish but that should be something that goes through review to make sure we're not creating bad experiences in other places this is used.